authors/apps/profiles/tests/test_profiles.py

```diff
@@ -93,6 +93,16 @@ def setUp(self):
         self.token = self.login.data['token']
         self.client.credentials(HTTP_AUTHORIZATION='Bearer ' + self.token)
 
```

Comment:
We could also test if an authenticated user can get user profiles

Reply:
> We could also test if an authenticated user can get user profiles

This has already been tested in `test_profiles.py` from line 96 to 104